appveyor.yml

```diff
@@ -61,7 +21,6 @@ build_script:
   - echo paths = ["."] > .cargo\config
   - git clone -q --depth 50 -b pending https://github.com/gtk-rs/examples _examples
   - cd _examples
-  - "echo Features: %FEATURES%"
-  - cargo build --features "%FEATURES%"
+  - cargo build --verbose
```

Comment:
The `echo` and `--verbose` are not necessary now but the `--features gtk_3_14` is.